cmd/agent/app/integrations.go

```diff
@@ -330,19 +349,52 @@ func pip(args []string) error {
 }
 
 func install(cmd *cobra.Command, args []string) error {
-	if err := validateArgs(args); err != nil {
+	if err := validateArgs(args, localWheel); err != nil {
 		return err
 	}
 
-	// Additional verification for installation
-	if len(strings.Split(args[0], "==")) != 2 {
-		return fmt.Errorf("you must specify a version to install with <package>==<version>")
+	cachePath, err := getTUFPipCachePath()
+	if err != nil {
+		return err
 	}
 
-	cachePath, err := getTUFPipCachePath()
+	here, err := executable.Folder()
 	if err != nil {
 		return err
 	}
+	constraintsPath := filepath.Join(here, relConstraintsPath)
+
+	pipArgs := []string{
+		"install",
+		"--constraint", constraintsPath,
+		"--cache-dir", cachePath,
+		// We replace the PyPI index with our own by default, in order to prevent
+		// accidental installation of Datadog or even third-party packages from
+		// PyPI.
+		"--index-url", tufIndex,
+		// Do *not* install dependencies by default. This is partly to prevent
+		// accidental installation / updates of third-party dependencies from PyPI.
+		"--no-deps",
+	}
+
+	if localWheel {
+		// Specific case when installing from locally available wheel
+		// No compatibility verifications are performed, just install the wheel (with --no-deps still)
+		// Verify that the wheel depends on `datadog_checks_base` to decide if it's an agent check or not
+		fmt.Println("Trying to install a package from a local file is at your own risk. No security verifications will be performed.")
+		fmt.Println("You should only use this to install an agent check, and not arbitrary wheels.")
```

Comment:
Maybe we could simplify the two lines with:
"For your security, only use this to install wheels containing an agent integration and coming from a known source. The Agent cannot perform any verification on local wheels."

Reply:
Sounds much better, thanks